unit_testing/cli/test_cli_sct_smooth_spinalcord.py

```diff
@@ -12,3 +14,10 @@ def test_sct_smooth_spinalcord_backwards_compat(script_runner):
     logger.debug(f"{ret.stderr}")
     assert ret.success
     assert ret.stderr == ''
+
+
+def test_sct_smooth_spinalcord_o_flag():
```

Comment:
Same comments re: subprocess and `tmp_path`.

Reply:
[Here](https://github.com/neuropoly/spinalcordtoolbox/blob/dc97d42730d7e603f83e54c68f242ad2bef7189b/spinalcordtoolbox/scripts/sct_smooth_spinalcord.py#L120-L127), it does not take into consideration if args are passed to main (only if they are passed through the sys)

Reply:
tmp_path fixed